src/main/kotlin/com/autonomousapps/Flags.kt

```diff
@@ -16,10 +16,17 @@ object Flags {
   private const val FLAG_MAX_CACHE_SIZE = "dependency.analysis.cache.max"
   private const val FLAG_TEST_ANALYSIS = "dependency.analysis.test.analysis"
   private const val FLAG_PRINT_BUILD_HEALTH = "dependency.analysis.print.build.health"
+  /**
+   * Android build variant to not analyze i.e.
+   *
+   * dependency.analysis.android.ignored_variants=release
+   */
+  private const val FLAG_ANDROID_IGNORED_VARIANTS = "dependency.analysis.android.ignored_variants"
```

Comment:
I think this should be `dependency.analysis.android.ignored.variants`. From experience, I know that having different word delimiters can be annoying.

Reply:
done